helm/fiaas-skipper/templates/rbac.yaml

```diff
@@ -92,3 +90,28 @@ roleRef:
   name: fiaas-controller
   apiGroup: rbac.authorization.k8s.io
 {{- end }}
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.name }}
+
+---
+kind: ClusterRoleBinding
+{{- if semverCompare ">=1.8" .Capabilities.KubeVersion.GitVersion }}
+apiVersion: rbac.authorization.k8s.io/v1
+{{- else }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+{{- end }}
+metadata:
+  name: {{ .Values.name }}
+subjects:
+- kind: ServiceAccount
+  name: {{ .Values.name }}
+  namespace: {{ .Release.Namespace }}
+roleRef:
+  kind: ClusterRole
+  name: fiaas-controller
```

Comment:
I think skipper technically doesn't need *all* the privileges of the fiaas-controller role - what do you think about creating a separate role for it?

Reply:
i want to eventually do that, maybe i should add it to this PR. But I also want to know exactly what privileges it needs